Find a functionally equivalent code:android.graphics.drawable.GradientDrawable.setGradientType:COMMENT
Method Modifier: public      
Comment:/**
 * <p>Sets the type of gradient used by this drawable..</p>
 * <p><strong>Note</strong>: changing this property will affect all instances
 * of a drawable loaded from a resource. It is recommended to invoke
 * {@link #mutate()} before changing this property.</p>
 *
 * @param gradient The type of the gradient: {@link #LINEAR_GRADIENT},
 * {@link #RADIAL_GRADIENT} or {@link #SWEEP_GRADIENT}
 *
 * @see #mutate()
 */

@@ -1,5 +1,5 @@
 {
     mGradientState.setGradientType(gradient);
-    mRectIsDirty = true;
+    mGradientIsDirty = true;
     invalidateSelf();
 }

